yasa-ui/src/style/iconfont/iconfont.js

```diff
@@ -0,0 +1,67 @@
+    v = (l = document.getElementsByTagName('script'))[l.length - 1].getAttribute('data-injectcss');
+  if (v && !c.__iconfont__svg__cssinject__) {
+    c.__iconfont__svg__cssinject__ = !0;
+    try {
+      document.write(
+        '<style>.svgfont {display: inline-block;width: 1em;height: 1em;fill: currentColor;vertical-align: -0.1em;font-size:16px;}</style>',
+      );
+    } catch (c) {
+      console && console.log(c);
+    }
+  }
+  function m() {
+    s || ((s = !0), e());
+  }
+  (a = function () {
+    let c,
+      l,
+      a,
+      t,
+      e,
+      o = document.createElement('div');
+    (o.innerHTML = i),
+      (i = null),
+      (c = o.getElementsByTagName('svg')[0]) &&
+        (c.setAttribute('aria-hidden', 'true'),
+        (c.style.position = 'absolute'),
+        (c.style.width = 0),
+        (c.style.height = 0),
+        (c.style.overflow = 'hidden'),
+        (l = c),
+        (a = document.body).firstChild
```

Comment:
Codacy found an issue: [Avoid assignments in operands](https://app.codacy.com/gh/yasa-org/yasa/pullRequest?prid=5815330)